django/river/api/views.py

```diff
@@ -50,12 +46,10 @@ def create(self, request):
         serializer = serializers.PreviewSerializer(data=request.data)
```

Comment:
I think some attributes are missing in this view (and it breaks the openapi schema generation). From the `GenericAPIView` doc:
```python
class GenericAPIView(views.APIView):
"""
    Base class for all other generic views.
    """
    # You'll need to either set these attributes,
    # or override `get_queryset()`/`get_serializer_class()`.
    # If you are overriding a view method, it is important that you call
    # `get_queryset()` instead of accessing the `queryset` property directly,
    # as `queryset` will get evaluated only once, and those results are cached
    # for all subsequent requests.
    queryset = None
    serializer_class = None
```